src/shared/ui/layouts/MainLayout/Footer.tsx

```diff
@@ -1,3 +1,21 @@
+import { Button } from "@/shared/ui";
+
 export function Footer() {
-  return <h1>Footer</h1>;
+  return (
+    <div className="m-2 flex justify-between gap-6 bg-gradient-to-r from-[#ae2896] to-[#4f9bf4] pb-2 pl-4 pr-6 pt-3">
```

Comment:
We should delete the `m-2` class. To be reusable, a component only needs to define its internal spacing